### PR TITLE
Recommended Song PORO

### PR DESCRIPTION
**Pull request by BrennanAyers**
This PR brings in a PORO for us to render RecommendedSongs correctly on the Dashboard view. Previously, we were just accessing the hash that we built, which is not Ruby Idiomatic.
This PORO is very simple, just taking in a Sifted Song and pulling the SpotifyID. That's it.

Also this brings back our requirement for rendering Charts in `application.js` of the `Chart.bundle.min` package.

### CODE REVIEW

File: app/facades/dashboard_show_facade.rb [hunk 1]
```diff
@@ -10,19 +10,21 @@ def initialize(current_user)
...
 
   def spotify_song_uris
-    recommended_songs.map { |song| 'spotify:track:' + song[:spotify_id] }.join(',')
+    recommended_songs.map { |song| 'spotify:track:' + song.spotify_id }.join(',')
   end
 
   def recommended_songs
     songs = @user.top_songs(5).map(&:spotify_id).join(',')
-    @recommended_songs ||= recommended_service.get_recommendations(songs)
+    @recommended_songs ||= recommended_service.get_recommendations(songs).map do |song|
```
[the remark this answers is not in the document]
Comment (BrennanAyers) [reply app/facades/dashboard_show_facade.rb]: 🤷‍♀

File: spec/models/recommended_song_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    data = {
+      album: 'FML',
+      album_art_url: 'https://i.scdn.co/image/f4df5c3409bb23c7170639a0132849c600594e24',
+      artist: 'K.Flay',
+      length: 207724,
+      spotify_id: '6kbCH9kQoNzaEt1R1rizpR',
+      spotify_url: 'https://open.spotify.com/track/6kbCH9kQoNzaEt1R1rizpR',
+      title: 'FML'
+    }
+    song = RecommendedSong.new(data)
```
Comment (alexander-mathieu) [inline spec/models/recommended_song_spec.rb]: Solid PORO test.

File: app/models/recommended_song.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Top Level Class Documentation
+class RecommendedSong
+  attr_reader :spotify_id
+
+  def initialize(data)
```
Comment (alexander-mathieu) [inline app/models/recommended_song.rb]: 👍

File: app/facades/dashboard_show_facade.rb [hunk 4]
```diff
@@ -10,19 +10,21 @@ def initialize(current_user)
   end
 
   def spotify_song_uris
-    recommended_songs.map { |song| 'spotify:track:' + song[:spotify_id] }.join(',')
+    recommended_songs.map { |song| 'spotify:track:' + song.spotify_id }.join(',')
   end
 
   def recommended_songs
     songs = @user.top_songs(5).map(&:spotify_id).join(',')
```
Comment (alexander-mathieu) [inline app/facades/dashboard_show_facade.rb]: Woof.

**Pull request merged**